runtime/bcutil/ClassFileOracle.cpp

```diff
@@ -502,9 +502,8 @@ ClassFileOracle::walkAttributes()
 			break;
 		}
 		case CFR_ATTRIBUTE_PermittedSubclasses: {
-			/* PermittedSubclasses verification is for Java 15 preview only. Don't record the attribute for other class versions
-			 * since it may be corrupt. */
-			if ((59 == _classFile->majorVersion) && (0 < _classFile->minorVersion)) {
+			/* PermittedSubclasses verification is for Java version >= 15 */
+			if (_classFile->majorVersion >= 59) {
```

Comment:
59 is Java 15, correct?  Shouldn't this be `>59` (Java 16) then?
```suggestion
			if (_classFile->majorVersion > 59) {
```

Reply:
Agreed and updated as suggested above (I previously thought it should include `59` given it was implemented in Java 15).